src/DatePicker/Utilities.elm

```diff
@@ -314,3 +317,69 @@ toUtcDateTimeString datetime =
         ++ ":"
         ++ addLeadingZero (Time.toSecond Time.utc datetime)
         ++ " (UTC)"
+
+
+setDayNotTime : Maybe Posix -> Posix -> Posix
```

Comment:
This function could use a test. I know we don't have any yet but we have to start somewhere :)